### PR TITLE
fix(core/cluster): Do not wrap long servergroup stack names; use ellipsis

### PR DESCRIPTION
**Pull request by christopherthielen**

### CODE REVIEW

File: app/scripts/modules/core/src/cluster/rollups.less [hunk 1]
```diff
@@ -182,20 +182,27 @@ table.instances {
...
     justify-content: space-between;
     background-color: var(--color-accent-g2);
     padding: 0 10px;
     border-right: 1px solid var(--color-alabaster);
     border-left: 1px solid var(--color-alabaster);
     font-weight: 600;
+    > div {
+      white-space: nowrap;
+      overflow-x: hidden;
+      text-overflow: ellipsis;;
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/cluster/rollups.less]: ;;

**Comment by anotherchrisberry** [comment]
one super minor thing but LGTM

File: app/scripts/modules/core/src/cluster/rollups.less (hunk 1 above)
Comment (christopherthielen) [reply app/scripts/modules/core/src/cluster/rollups.less]: Ok I’ll add the third semicolon;;;

Comment (christopherthielen) [reply app/scripts/modules/core/src/cluster/rollups.less]: Ok I’ll add the third semicolon;;;

Comment (jrsquared) [reply app/scripts/modules/core/src/cluster/rollups.less]: He's going to add a third semicolon apparently

Comment (christopherthielen) [reply app/scripts/modules/core/src/cluster/rollups.less]: ;;;

**Pull request merged**